pkg/auth/auth_oidc_test.go

```diff
@@ -117,16 +120,27 @@ func (m *mockOIDCProvider) handleToken(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
-		if r.Form.Get("refresh_token") != testValidRefreshToken {
+		// compiling the regexp might add some latency that can be useful for
+		// benchmarking
+		refreshTokenRegEx := regexp.MustCompile(`^` + testValidRefreshToken + `([0-9]*)`)
```

Comment:
nit:
lets put the `refreshTokenRegEx`  into `init()` function of this package

Reply:
in the worst case we could add it to the http server init but as the comment explains - no need to optimize